config-files/usr/share/plasma/desktoptheme/openSUSElight/metadata.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=openSUSElight
-Comment=Default openSUSE theme
+Comment=light openSUSE theme
```

Comment:
`Light`

Reply:
Yes you are right. I removed all icons so that themes can inherit from upstream (breeze-light and breeze-dark). 
The svgz I added back:
opensuse start.sgvz , in the icons folder. 
indicators.svgz , in the icontasks folder. This one was already in opensuse themes, it looks good so I believe we can keep it. If you prefer to remove it, please let me know.

![start-svgz](https://user-images.githubusercontent.com/6520987/68827915-a3de9080-06de-11ea-96b5-27c4a4f1b3db.png)
![indicators-svgz](https://user-images.githubusercontent.com/6520987/68827919-aa6d0800-06de-11ea-99ed-09da479a360f.png)